src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -212,6 +208,40 @@ class MediaBaseViewer extends BaseViewer {
     }
 
     /**
+     * Handler for autoplay
+     *
+     * @private
+     * @emits autoplay
+     * @return {void}
+     */
+    handleAutoplay() {
+        const isAutoplayEnabled =
+            this.cache.has(MEDIA_AUTOPLAY_CACHE_KEY) && this.cache.get(MEDIA_AUTOPLAY_CACHE_KEY) === 'Enabled';
+        this.emit('autoplay', isAutoplayEnabled);
+    }
+
+    /**
+     * Determines if media should autoplay based on cached settings value.
+     *
+     * @private
+     * @emits volume
+     * @return {void}
+     */
+    checkAutoplay() {
+        if (
+            (!this.cache.has(MEDIA_AUTOPLAY_CACHE_KEY) || this.cache.get(MEDIA_AUTOPLAY_CACHE_KEY) === 'Disabled') &&
```

Comment:
This code implies if nothing is set in the cache, then don't autoplay. Is that what you want? (I think so?)

Reply:
Yep, I'll add a clarifying comment.